corehq/apps/data_dictionary/util.py

```diff
@@ -238,6 +238,7 @@ def fields_to_validate(domain, case_type_name):
 @quickcache(['domain', 'case_type'], timeout=24 * 60 * 60)
 def get_gps_properties(domain, case_type):
     return set(CaseProperty.objects.filter(
+        deprecated=False,
```

Comment:
Why exclude deprecated properties? Just because they are deprecated doesn't mean that won't exist in case data.

Reply:
Mostly it was because I got confused - the GPS property in the bug that triggered this was deprecated, so I didn't see it on the DD page until I realized I needed to display those too.  I was thinking of those as properties that the project no longer cares about, so it's senseless to do anything with them.  That said, in this case there's no real downside to indexing deprecated properties, and that _would_ ease things if a project toggles something off and then back on later.